packages/format-library/src/underline/index.js

```diff
@@ -33,6 +34,15 @@ export const underline = {
 					character="u"
 					onUse={ onToggle }
 				/>
+				<RichTextToolbarButton
```

Comment:
We might want to explore whether we could use some sort of platform guard which would conditionally include this component only for mobile apps. Similar to what they do in `react-native-web`:
https://github.com/necolas/react-native-web/blob/0e81c6ef2758d4ca9b2099b1d04a4f8c417f0f43/packages/react-native-web/src/exports/Platform/index.js

I think @koke shared this idea some time ago. It would be a great opportunity to test it in action. I know that you use the native version directly from `react-native` library. We could expose it from `@wordpress/element`.

cc @youknowriad, @hypest and @Tug